Jenkinsfile.tag

```diff
@@ -0,0 +1,9 @@
+// Load a library dynamically. For more detail, see:
+// https://jenkins.io/doc/book/pipeline/shared-libraries/#defining-declarative-pipelines
+library identifier: 'crossplane-cicd@master', retriever: modernSCM(
+  [$class: 'GitSCMSource',
+   remote: 'https://github.com/crossplane/cicd',
+   credentialsId: 'github-upbound-jenkins'])
+
+// This is the name of a file declared in vars/{{fileName}}.groovy in the library repository.
+runStackTagPipeline()
```

Comment:
I don't think we should re-use stack pipelines in this repo as it's not a stack.